src/components/molecule_word_card/index.shared.tsx

```diff
@@ -12,6 +12,7 @@ import { PageQueryConst } from '@/constants/page-queries.constant'
 import StyledCountdownTimer from '@/atoms/StyledCountdownTimer'
 import TagButtonLanguage from '../atom_tag_chip/index.language'
 import TagChipCustomized from '../atom_tag_chip/index.customized'
+import { useWindowSize } from 'react-use'
```

Comment:
dont forget to delete

Reply:
Handled in [65825d0](https://github.com/ajktown/wordnote/pull/205/commits/65825d09cd61c7debbdc17e9cffc23c128200fb6)